recipe/meta.yaml

```
@@ -81,10 +86,17 @@ requirements:
 # It seems that the apply_parallel tests can have relatively small errors on windows
 {% set tests_to_skip = tests_to_skip + " or test_apply_parallel_rgb" %}  # [win]
 
+# skip known failure on Windows PyPy builds
+{% set tests_to_skip = tests_to_skip + " or test_hough_ellipse_zero_angle or test_hough_ellipse_non_zero" %}  # [win and (python_impl == 'pypy')]
+
 test:
   requires:
     - pytest
     - pytest-localserver
+    # 2021/12/04: 0.19.0: hmaarrfk
+    # We noticed that osx had strange failures with cythonized code
+    # Testing with full datasets on OSX to ensure we don't have bugs
+    - pooch  # [osx]
```

```suggestion
    - pooch
```

Can we run a single round of full tests before merging?

---

I see that pooch was already listed in `requirements/test.txt` in the main repo, so having this unconditionally seems consistent with that.